dvc/tree/dvc.py

```diff
@@ -103,14 +103,14 @@ def exists(self, path):  # pylint: disable=arguments-differ
         try:
             self.metadata(path)
             return True
-        except OutputNotFoundError:
+        except FileNotFoundError:
```

Comment:
Could've made this a separate change, but in new code it felt too weird that I have to catch an internal exception.